crates/sui-graphql-rpc/src/types/address.rs

```diff
@@ -145,12 +146,21 @@ impl Address {
     // TODO disabled-for-rpc-1.5
     // pub async fn name_service_connection(
     //     &self,
-    //     ctx: &Context<'_>,
     //     first: Option<u64>,
     //     after: Option<String>,
     //     last: Option<u64>,
     //     before: Option<String>,
     // ) -> Result<Option<Connection<String, NameService>>> {
     //     unimplemented!()
     // }
+
+    pub async fn dynamic_field_connection(
+        &self,
+        _first: Option<u64>,
+        _after: Option<String>,
+        _last: Option<u64>,
+        _before: Option<String>,
+    ) -> Result<Option<Connection<String, DynamicField>>> {
+        Err(crate::error::Error::DynamicFieldOnAddress.extend())
+    }
```

Comment:
I'm pretty sure you can have the `Result` type return anything that can be extended, so you can write this like so:

```suggestion
    pub async fn dynamic_field_connection(
        &self,
        _first: Option<u64>,
        _after: Option<String>,
        _last: Option<u64>,
        _before: Option<String>,
    ) -> Result<Option<Connection<String, DynamicField>>, Error> {
        Err(Error::DynamicFieldOnAddress)
    }
```

Reply:
I see, that is preferable indeed, thanks!